editor/editor_inspector.cpp

```diff
@@ -2662,6 +2662,17 @@ void EditorInspector::update_tree() {
 				}
 			}
 
+			// Process group names that are created by a "/" in the property name.
+			// Groups and subgroups added with ADD_GROUP or ADD_SUBGROUP do not have their names processed.
+			if (path.contains("/")) {
+				String processed_path = "";
+				Vector<String> slices = path.split("/");
+				for (int i = 0; i < slices.size() - 1; i++) {
+					processed_path += EditorPropertyNameProcessor::get_singleton()->process_name(slices[i]) + "/";
+				}
+				path = processed_path + slices[slices.size() - 1];
+			}
```

Comment:
Filtering expects `path` to contain the original property path instead of processed ones. So I think you have to keep the process logic where it was.

You can tell if it needs processing (i.e. it's a hint string from `PROPERTY_HINT_GROUP` / `PROPERTY_HINT_SUBGROUP` ) by comparing with current `group` / `subgroup`.

Also, it still needs to go through `TTR` even it's a pre-capitalized group name.

Reply:
Ok, thanks. I never thought of doing that; that's a much better solution.